supplementary_style_guide/style_guidelines/code-commands.adoc

```diff
@@ -20,8 +21,20 @@ Do not rely solely on a shell prompt in a sample command to indicate the require
 If you include a shell prompt to indicate that a user with root privileges must run the command, also include a statement about this requirement in the step text, the introductory text, or the prerequisites.
 ====
 +
-* If a command requires a temporary switch to root privileges, use `sudo` at the beginning of the sample command syntax. If the sample command includes a shell prompt together with `sudo`, the prompt should remain `$` and not be changed to `#`.
-* If multiple commands in a procedure require root privileges, add introductory content to tell the user what to do. For example, "The commands in this procedure require root privileges to run. Either run `su -` to switch to the root user or preface the commands with `sudo`."
+* If a sample command includes a shell prompt together with `sudo`, do not change the prompt to `#`. Keep the `$` prompt, as shown in the following example: 
```

Comment:
```suggestion
* Use the `$` prompt, not `#`, when a sample command includes `sudo`, as shown in the following example: 
```

Reply:
Great suggestion. Made the change with a slight revision.Thanks.